resources/dto/dtoWeather.go

```diff
@@ -0,0 +1,73 @@
+package dto
```

Comment:
Did you write those models yourself? I always love to check if there is something ready to use out there I can use so I can go have a coffee more often :rofl: Since you are using [https://www.weatherapi.com/](https://www.weatherapi.com/) In the 'Getting started" section there is convenient reference "We also have SDK for popular framework/languages [available on Github for quick integrations](https://github.com/weatherapicom/)." I suggest you import the referenced go package and use the models from there.